modules/search/instant-search/components/search-results.jsx

```diff
@@ -4,33 +4,17 @@
  * External dependencies
  */
 import { __, _n, sprintf } from '@wordpress/i18n';
-import { h, Component } from 'preact';
+import { h, Component, Fragment } from 'preact';
```

Comment:
Is the `Fragment` import necessary here?

Reply:
No - good catch. I'll used to the linter picking up things like that on Calypso!